modin/engines/base/frame/partition_manager.py

```diff
@@ -238,6 +245,8 @@ def broadcast_axis_partitions(
         right : The right partitions.
         keep_partitioning : boolean. Default is False
             The flag to keep partitions for Modin Frame.
+        apply_indices : list(int),
```

Comment:
Is apply_indices really a list of ints here?

Reply:
Yes, it is. 

[`broadcast_apply_full_axis`](https://github.com/dchigarev/modin/blob/a44e9a607a6e3710e86f0c8a66980316e6f11502/modin/engines/base/frame/data.py#L1630) obtains `apply_indices` as a list of labels and [converts](https://github.com/dchigarev/modin/blob/a44e9a607a6e3710e86f0c8a66980316e6f11502/modin/engines/base/frame/data.py#L1673-L1675) it to numeric indices. So yeah, `apply_indices` is a list of ints here.

Reply:
But yeah, doc string doesn't tell us that this parameter is optional, fixed it in the last commit